package.json

```diff
@@ -1,58 +1,67 @@
 {
-    "name": "snapsvg",
-    "version": "0.5.1",
-    "description": "JavaScript Vector Library",
-    "main": "./dist/snap.svg.js",
-    "repository": {
-        "type": "git",
-        "url": "git@github.com:adobe-webplatform/Snap.svg.git"
+  "name": "snapsvg",
+  "version": "0.5.1",
+  "description": "JavaScript Vector Library",
+  "main": "./dist/snapsvg.js",
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:adobe-webplatform/Snap.svg.git"
+  },
+  "author": "Dmitry Baranovskiy",
+  "license": "Apache-2.0",
+  "dependencies": {
+    "eve": "~0.5.1"
+  },
+  "scripts": {
+    "clean": "rimraf dist",
```

Comment:
It's a habit for me to have tasks that delete any existing build artifacts prior to running a bundle. That way there's no possibility of an older version hanging around prior to a publication. If it's not useful to your workflow though, I can get rid of it.  I use rimraf because it's cross-platform and won't emit an error on ENOENTs.